tests/qpu-test-files/update_aspen_qpu_files.py

```diff
@@ -18,12 +19,33 @@ def device_to_chipspec(device, version=0.0, timestamp=None):
                 specs=device.specs.to_dict())
 
 
+# Device/Date/Lattice.qpu
+qpu_dir = "{}/{}/{}.qpu"
+
+
 def main():
-    now = datetime.datetime.today()
-    for device in map(get_lattice, list_quantum_computers(qvms=False)):
-        print(device.name, file=sys.stderr)
-        with open(f'{device.name}.qpu', 'w') as f:
-            json.dump(device_to_chipspec(device, timestamp=now), f, indent=2)
+    now = str(datetime.datetime.today())
+    print(list_devices())
```

Comment:
```suggestion
    print(list_devices(), file=sys.stderr)
```

for pleasing symmetry with the `lattice.name` printing below.  or else remove if was a debugging thing